os_migrate/roles/import_workloads/tasks/workload.yml

```diff
@@ -47,5 +47,9 @@
     when: prelim.changed
 
   rescue:
-    - debug:
+    - name: give some time to debug this!
```

Comment:
Should the wait_for go into the e2e test instead -- let the ansible playbook fail as normal, and if it fails, we keep the env around for some time?

I think we should keep the change from 'debug' to 'fail' here though. The rescue+debug pattern could result in unnoticed errors?